.github/workflows/docs.yml

```diff
@@ -1,5 +1,11 @@
 name: Docs
-on: [push]
+on:
+  pull_request:
```

Comment:
I don't think you want the `pull_request` bit. What it says (IIRC) is to run when a pull request is opened that targets the `main` branch. I think you just want the `push` part.